api.js

```diff
@@ -180,6 +186,25 @@ Api.prototype._prefixTitle = function (file) {
 	return prefix;
 };
 
+Api.prototype._restartTimer = debounce(function () {
```

Comment:
Also, you shouldn't use `debounce` on a prototype method, as it will debounce across every instance and only call on the most recent.

https://github.com/component/debounce/issues/8